README.md

```diff
@@ -46,7 +54,7 @@ ISD is built to make it easy to collect the most important metrics from your cri
 
 ## Custom Plugin Scripts
 
-Instrumental Daemon can monitor arbitrary processes and system events through a plugin scripting system. Writing plugins is easier than you'd think! Plugin script installation and development instructions are listed in [PLUGIN_SCRIPTS.md](PLUGIN_SCRIPTS.md).
+ISD can monitor arbitrary processes and system events through a plugin scripting system. Writing plugins is easier than you'd think! Plugin script installation and development instructions are listed in [PLUGIN_SCRIPTS.md](PLUGIN_SCRIPTS.md).
```

Comment:
I could see making an explicit link to each of the samples here, since they are neat.